python/example.py

```diff
@@ -29,5 +29,25 @@
     # Get reader
     reader = dremio_flight_endpoint.get_reader(flight_client)
 
-    # Print out the data as a dataframe
-    print(reader.read_pandas())
+    # OPTION 1: Read all data at once (suitable for smaller datasets)
+    # Uncomment this line if working with manageable data sizes.
+    # print(reader.read_pandas())
+
+    # OPTION 2: Read data in chunks using RecordBatchReader (recommended for large datasets)
+    # Uncomment the following block if working with large datasets
+
+    """
+    # Convert the reader to a RecordBatchReader to read data in chunks
+    record_batch_reader = reader.to_reader()
+
+    # Iterate through each RecordBatch and convert to pandas DataFrame
+    try:
+        for record_batch in record_batch_reader:
+            # Convert the current RecordBatch to a pandas DataFrame
+            df = record_batch.to_pandas()
```

Comment:
For ref, without chunks, it's possible to receive `grpc_status` 2, meaning unknown as the stream has been removed (see https://grpc.github.io/grpc/core/md_doc_statuscodes.html).
The problem can also happen when the flight service uses HTTP2 but it's façade by a proxy (ingress) that does HTTP1.1 (it can be "forced" using the `app-protocol` header).

So, this PR is correct, as a workaround. Without changing the client (no chunks), we have to play with the `app-protocol` annotation I guess.